ads

```diff
@@ -0,0 +1,556 @@
+=> [[33m:autosave_associated_records_for_shopping_cart[0m,
```

Comment:
co to za plik?

Reply:
Nie wiem jakim cudem tego nie zauważyłem. Czasami zwiesza mi się terminal przy pry, musiałem nieumyślnie zapisać log z console. Już naprawione, będę pilnować żeby nic podobnego więcej nie przeszło - i na pewno nie do code review.

Reply:
poczytaj o global gitignore :v: